hikari/internal/cache.py

```diff
@@ -619,7 +619,7 @@ class MentionsData(BaseData[messages.Mentions]):
     """A model for storing message mentions data in an in-memory cache."""
 
     users: undefined.UndefinedOr[typing.Mapping[snowflakes.Snowflake, RefCell[users_.User]]] = attr.ib()
-    role_ids: undefined.UndefinedOr[typing.Sequence[snowflakes.Snowflake]] = attr.ib()
+    role_ids: undefined.UndefinedOr[typing.Tuple[snowflakes.Snowflake, ...]] = attr.ib()
```

Comment:
Why all these changes to Tuples? They should be Sequences

Reply:
Because they are specifically tuples, putting sequences for these few was a mistake since that allows for a mutable list to be used but in the cache but we want them to be specifically immutable